src/common/StatementMetadata.cpp

```diff
@@ -167,7 +167,7 @@ const char* StatementMetadata::getPlan(bool detailed)
 
 	if (plan->isEmpty())
 	{
-		UCHAR info[] = {detailed ? isc_info_sql_explain_plan : isc_info_sql_get_plan};
```

Comment:
Try (for this particular item) the following:


diff --git a/src/jrd/inf_pub.h b/src/jrd/inf_pub.h
index 9c3b0c2..c223ec4 100644
--- a/src/jrd/inf_pub.h
+++ b/src/jrd/inf_pub.h
@@ -417,7 +417,7 @@ enum info_db_provider
 #define isc_info_sql_records             23
 #define isc_info_sql_batch_fetch         24
 #define isc_info_sql_relation_alias            25
-#define isc_info_sql_explain_plan              26
+#define isc_info_sql_explain_plan              26u
 #define isc_info_sql_stmt_flags         27
 
 /*********************************/